core/lib/Cardano/Script.hs

```diff
@@ -31,6 +31,8 @@ module Cardano.Script
 
 import Prelude
 
+import Codec.Binary.Encoding
+    ( AbstractEncoding (..), encode, fromBase16 )
```

Comment:
I forgot to add that I don't really like module names like this.
It confuses me because it looks like the module comes from another package.
Given that it deals with bech32/base56, it is very much a module which should be called `Cardano.Addresses.Encoding`.

Also, this module `Cardano.Script`. There are multiple kinds of scripts in cardano. How about `Cardano.Addresses.Script`?

Reply:
sound idea -> https://github.com/input-output-hk/cardano-addresses/pull/84/commits/b899a28d6c6cd517bdf854bc7a75b8165cb7a1e2